eo-maven-plugin/src/main/java/org/eolang/maven/Home.java

```diff
@@ -174,33 +181,46 @@ public String rel(final Path file) {
     /**
      * Check if exists.
      *
-     * @param path Path
+     * @param path Cwd-relative path to file
      * @return True if exists
      */
     public boolean exists(final Path path) {
-        return Files.exists(this.path(path));
+        return Files.exists(this.absolute(Home.clean(path)));
     }
 
     /**
      * Load bytes from file by path.
      *
-     * @param path Path to the file
+     * @param path Cwd-relative path to file
      * @return Bytes of file
      * @throws IOException if method can't find the file by path or
      *  if some exception happens during reading the file
      */
     public Bytes load(final Path path) throws IOException {
-        return new BytesOf(Files.readAllBytes(this.path(path)));
+        return new BytesOf(Files.readAllBytes(this.absolute(Home.clean(path))));
     }
 
     /**
-     * Path modification.
+     * Clean path.
      *
      * @param path Path
-     * @return Modified path (without bad symbols)
+     * @return Clean path
+     * @todo #1352:30min Move utility `clean` method out of `Home` class. Create
+     *  maybe separate class for this. Update all usages of this method.
      */
-    private static Path path(final Path path) {
+    @SuppressWarnings("PMD.ProhibitPublicStaticMethods")
+    public static Path clean(final Path path) {
```

Comment:
@volodya-lombrozo Not at this place as the puzzle above says.

Reply:
@mximp I mean that I don't clearly understand the case where we need to `clean` the path. Even if some path 
"%c5c5" it will be converted finally to the same path "%c5c5" (not the **업**).  I've tried to run tests with different strings encodings (Windows-1252 and ASCII) and result is the same. What am I missing?

Reply:
If we have ASCII encoded spring, we will receive some garbage like `????` even if we convert it to  UTF-8

Reply:
@volodya-lombrozo Indeed: this method seems to apply none. All tests are still green if I remove it...
I suggest to deal with this as part of a puzzle for this method.